code/modules/surgery/organs/tongue.dm

```diff
@@ -190,6 +190,17 @@
 	attack_verb = list("beeped", "booped")
 	taste_sensitivity = 25 // not as good as an organic tongue
 
+/obj/item/organ/tongue/robot/emp_act(severity)
+	switch(severity)
+		if(1)
+			owner.apply_effect(STUTTER, 60)
+			owner.emote("scream")
+			to_chat(owner, "<span class='warning'>Alert: Vocal cords are malfunctioning.</span>")
+		if(2)
+			owner.apply_effect(STUTTER, 30)
+			owner.emote("scream")
+			to_chat(owner, "<span class='warning'>Alert: Vocal cords are malfunctioning.</span>")
```

Comment:
no point having these two lines duplicate in each. If it'll happen regardless, stick it outside of the switch.

Reply:
You know, I could've sworn there was a reason I did it this way. But it turns out it works without the switch. fixed.